server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequest.java

```diff
@@ -184,6 +198,9 @@ public ActionRequestValidationException validate() {
             if (order != SortOrder.ASC) {
                 validationException = addValidationError("can't use non-default sort order with verbose=false", validationException);
             }
+            if (policies.length != 0) {
+                validationException = addValidationError("can't use slm policy filter with verbose=false", validationException);
+            }
```

Comment:
Can we add a generic validation to not use `-_none` and perhaps even not allow `_` or `-_` prefixed policies?

Reply:
`-_none` "theoretically" you could have a policy by that name I think :D